src/test/java/org/opensearch/knn/TestUtils.java

```diff
@@ -15,14 +15,12 @@
 import org.opensearch.common.xcontent.NamedXContentRegistry;
 import org.opensearch.common.xcontent.XContentFactory;
 import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.knn.index.SpaceType;
 import org.opensearch.knn.index.codec.KNNCodecUtil;
-
-import java.io.BufferedReader;
-import java.io.FileReader;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+import org.opensearch.knn.plugin.script.KNNScoringUtil;
+import java.io.*;
```

Comment:
There is some debate around whether wildcard imports should be used. In general, I think we should avoid them. Google style follows this convention as well: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports. 

Here is some discussion: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
I was not aware of this. Thanks for sharing it. updated the code.